packages/DeadCode/src/Rector/FunctionLike/RemoveDeadReturnRector.php

```diff
@@ -73,7 +73,7 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
-        $lastStmt = $node->stmts[count($node->stmts) - 1];
+        $lastStmt = array_values(array_slice($node->stmts, -1))[0];
```

Comment:
Could you decouple it to 2 lines with some variable names? It's very hard to read, too many things at once